content/doc/book/scaling/scaling-jenkins-on-kubernetes.adoc

```diff
@@ -199,13 +199,13 @@ On the other hand, if you don’t have unique requirements for your agents, you
 Kubernetes Pod Template Name - can be any and will be shown as a prefix for unique generated agents' names, which will be run automatically during builds
 Docker image - the docker image name that will be used as a reference to spin up a new Jenkins agents.
 
-image::pod-template-configuration.png[pod-template-configuration]
+image::Pod-template.png[pod-template-configuration]
 
-== Using Jenkins Agents
+== Using Jenkins Agentsgit 
```

Comment:
```suggestion
== Using Jenkins Agents
```
Removal of undesired text.

Reply:
ok will do that thanks for your help I really appreciate it